orchestra/todos/urls.py

```diff
@@ -13,7 +13,8 @@
     url(r'^todo/$',
         TodoViewset.as_view({'post': 'create', 'get': 'list'}), name='todos'),
     url(r'^todo/(?P<pk>[0-9]+)/$',
-        TodoViewset.as_view({'put': 'update'}), name='todo'),
+        TodoViewset.as_view(
```

Comment:
sanity-checking what this is for?

Reply:
We are 'retrofitting' `TodoViewset` to the old URLs :)
Applied rest_framework's router.